src/org/usfirst/frc/team694/robot/RobotMap.java

```diff
@@ -65,5 +59,6 @@
     public static final int CRAB_ARM_LEFT_MOTOR_PORT = -1;
     public static final int CRAB_ARM_RIGHT_MOTOR_PORT = -1;
     public static final int GRABBER_SOLENOID_PORT = -1;
```

Comment:
These are solenoid ports, not Lift Motor ports.

Either move these underneath a "Solenoid" section or rename the "Lift Motor Ports" section to just be called "Lift Ports"